.github/workflows/sonarcloud.yml

```
@@ -1,86 +1,67 @@
+          # Set the base directory for the Sonar analysis (where gradlew is)
+          projectBaseDir: project/backend
+          # args: > # No args needed here! Plugin gets config from build.gradle/properties
+            # -Dsonar.projectKey=... # Define in build.gradle or gradle.properties
+            # -Dsonar.organization=... # Define in build.gradle or gradle.properties
+            # -Dsonar.host.url=https://sonarcloud.io # Define in build.gradle or gradle.properties
         env:
+          # Define the SonarCloud token in your GitHub repository secrets
           SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          # GITHUB_TOKEN is automatically provided by Actions, required for PR decoration
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)